src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -283,6 +283,8 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
     postInit() {
         super.postInit();
 
+        this.gpuProfiler = new WebgpuGpuProfiler(this);
```

Comment:
Should profiler always be created? For one thing this'll mean we cannot tree shake it.